### PR TITLE
First pass generate extra files for back compat

### PR DESCRIPTION
**Pull request by ericanderson**
Broken into two commits, one for the code i edited and one for the generated code.

### CODE REVIEW

File: packages/generator/src/v1.1/generateIndexFile.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ export async function generateIndexFile(fs: MinimalFs, outDir: string) {
       Unauthorized,
       UnknownError, 
     } from '@osdk/legacy-client';
-    export { Ontology } from "./Ontology";
+    export type { Ontology } from "./Ontology";
```
[the remark this answers is not in the document]
Comment (ericanderson) [reply packages/generator/src/v1.1/generateIndexFile.ts:92]: The way we export the type covers this as in place we evaluate it. And the foundryclient is what uses it directly, not any client code. So I am exporting the type only so we dont further leak implementation details

[the remark this answers is not in the document]
Comment (ericanderson) [reply packages/generator/src/v1.1/generateIndexFile.ts:92]: The Todo app still works fine

File: packages/generator/src/v1.1/backcompat/generateOntologyRuntimeDistDir.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    "dist",
+  );
+  await fs.mkdir(runtimeDistDir, { recursive: true });
+
+  await generateOntologyProviderDir(fs, runtimeDistDir);
+  await generateAggregationsDir(fs, runtimeDistDir);
+
+  await fs.writeFile(
+    path.join(runtimeDistDir, "index.ts"),
+    // TRASHFIXME
```
Comment (mfedderly) [inline packages/generator/src/v1.1/backcompat/generateOntologyRuntimeDistDir.ts:41]: Do you intend to remove these markers and clean up the commented out code?

**Review by mfedderly — APPROVED** [review]

File: packages/generator/src/v1.1/backcompat/generateOntologyRuntimeDistDir.ts (hunk 2 above)
Comment (ericanderson) [reply packages/generator/src/v1.1/backcompat/generateOntologyRuntimeDistDir.ts:41]: The markers were for you or me in the future. This does not complete the job

**Pull request merged**